Streetcode/Streetcode.XUnitTest/MediatRTests/Media/Audio/GetAudioByIdTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore.Query;
+using Moq;
+using Streetcode.BLL.DTO.Media;
+using Streetcode.BLL.Mapping.Media;
+using Streetcode.BLL.MediatR.Media.Audio.GetById;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+using Entities = Streetcode.DAL.Entities.Media;
+
+namespace Streetcode.XUnitTest.MediatRTests.Media.Audio
+{
+    public class GetAudioByIdTests
+    {   
+        [Theory]
+        [InlineData(2)]
+        public async Task GetAudioById_ExistingId_Succcess(int id)
+        {
+            var repository = new Mock<IRepositoryWrapper>();
+            var mockMapper = new Mock<IMapper>();
+
+            var testAudio = new Entities.Audio() { Id = id };
+
+            repository.Setup(repo => repo.AudioRepository
+            .GetFirstOrDefaultAsync(It.IsAny<Expression<Func<Entities.Audio, bool>>>(), It.IsAny<Func<IQueryable<Entities.Audio>, IIncludableQueryable<Entities.Audio, Entities.Audio>>?>()))
+            .ReturnsAsync(testAudio);
```

Comment:
If we use it few times consider moving it to separate create method

Reply:
done